scripts/upload-modern-slavery-statement.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+"""
+Usage:
+    scripts/upload-modern-slavery-statement.py
+        <file_path> <supplier_id> <framework> <s3_file_name> [<stage>]
+
+Options:
+    -h --help   Show this screen.
+
+Parameters:
+    <file_path>    Local path of modern slavery statement file.
+    <supplier_id>  Supplier ID
+    <framework>    Framework
+    <s3_file_name> The file name that users will see. For example, 'modern-slavery-statement-2020-11-19.pdf'
+    <stage>        Stage. Defaults to production.
+"""
+from dmapiclient import DataAPIClient
+from docopt import docopt
+from dmutils.env_helpers import get_api_endpoint_from_stage
+from dmutils.s3 import S3
+import sys
+
+sys.path.insert(0, '.')
+from dmscripts.helpers.auth_helpers import get_auth_token
+
+
+def get_slavery_statement_declaration_key(declaration):
+    if declaration['declaration']['modernSlavery']:
+        return 'modernSlavery'
+
+    return 'modernSlaveryStatementOptional'
+
+
+if __name__ == '__main__':
+    arguments = docopt(__doc__)
+
+    file_path = arguments['<file_path>']
+    supplier_id = arguments['<supplier_id>']
+    framework = arguments['<framework>']
+    s3_file_name = arguments['<s3_file_name>']
+    stage = arguments['<stage>'] or 'production'
+
+    bucket_name = f"digitalmarketplace-documents-{stage}-{stage}"
+    s3_key_name = f"{framework}/documents/{supplier_id}/{s3_file_name}"
+    asset_url = f"https://www.digitalmarketplace.service.gov.uk/suppliers/{s3_key_name}"
```

Comment:
Are we sure this is correct?
The format in the example in the manual is:
 ```
"https://assets.digitalmarketplace.service.gov.uk/<framework_slug>/documents/<int:supplier_id>/<filename>"
```

Reply:
Oops. Thanks for noticing. Fixed.